akka-docs/src/test/scala/docs/stream/operators/sourceorflow/Fold.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sourceorflow
+
+//#imports
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.Source
+
+//#imports
+object Fold extends App {
+
+  //#histogram
+  case class Histogram(low: Long = 0, high: Long = 0) {
+    def add(i: Int): Histogram = if (i < 100) copy(low = low + 1) else copy(high = high + 1)
+  }
+  //#histogram
+
+  //#fold
+  implicit val sys = ActorSystem()
```

Comment:
we shouldn't show creation of new ActorSystem, since it will be copied and surprise..surprise..not very good